crates/aptos-push-metrics/src/lib.rs

```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
             "Starting push metrics loop. Sending metrics to {} with a frequency of {} seconds",
             push_metrics_endpoint, push_metrics_frequency_secs
         );
+
+        info!(
+            "Execution dashboard link: {} ",
+            Self::get_dashboard_link(chain_name, namespace)
+        );
         Some(thread::spawn(move || {
             Self::worker(
                 quit_receiver,
                 push_metrics_endpoint,
                 push_metrics_frequency_secs,
                 push_metrics_api_token,
+                push_metrics_extra_labels,
             )
         }))
     }
 
+    fn get_dashboard_link(chain_name: &str, namespace: &str) -> Url {
+        let end_time = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis()
+            .to_string();
+        let start_time = SystemTime::now()
+            .sub(Duration::from_secs(600))
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis()
+            .to_string();
+        let mut url =
+            Url::parse("https://o11y.aptosdev.com/grafana/d/execution/execution").unwrap();
+        url.query_pairs_mut()
+            .append_pair("from", &start_time)
+            .append_pair("to", &end_time)
+            .append_pair("var-Datasource", "VictoriaMetrics Global")
+            .append_pair("var-metrics_source", "All")
+            .append_pair("var-chain_name", chain_name)
+            .append_pair("var-cluster", "All")
+            .append_pair("var-namespace", namespace)
+            .append_pair("var-role", "All");
+
+        url
+    }
+
+    fn push_metrics_extra_labels(chain_name: &str, namespace: &str) -> Vec<String> {
+        vec![
+            format!("chain_name={}", chain_name),
+            "cluster=unknown".into(),
+            "metrics_source=unknown".into(),
+            "kubernetes_pod_name=unknown".into(),
```

Comment:
will these mess up with existing usage of the pusher? (The db-backup for example, where these are set from outside)